cmd/token/list.go

```diff
@@ -76,3 +77,14 @@ func tokenListCmd() *cobra.Command {
 	cmd.PersistentFlags().AddFlagSet(config.GetPersistentFlagSet())
 	return cmd
 }
+
+func checkListTokenRole(cmd *cobra.Command, args []string) error {
+	if listTokenRole == "" {
```

Comment:
Empty role IS actually supported in list command. When no role is given it lists both controller and worker tokens.